cmd/atlascmd/schema.go

```diff
@@ -31,15 +39,22 @@ var (
 		Long:  "The `atlas schema` command groups subcommands for working with Atlas schemas.",
 	}
 
+	// SchemaFlags are common flags used by schema commands.
+	SchemaFlags struct {
+		URL     string
+		Schemas []string
+
+		// Deprecated: DSN is an alias for URL.
+		DSN string
```

Comment:
Why it's here? We should not accept it in the project file.

Reply:
I moved flags common between inspect and apply here. It's not supported in the project file